editor/src/components/canvas/remix/utopia-remix-root-component.tsx

```diff
@@ -27,7 +27,7 @@ interface RemixNavigationContext {
 }
 
 interface RemixNavigationAtomData {
-  [pathString: string]: RemixNavigationContext
+  [pathString: string]: RemixNavigationContext | undefined
```

Comment:
Is this so that you can assign `undefined` values against a key? If that means something different to a key not existing I'd recommend putting some other marker value instead as `undefined` is the result you'd get for a non-existent index against an object.

Reply:
I added this so that null checks are enforced when looking up values from `RemixNavigationAtom`